redis/src/connection.rs

```diff
@@ -201,6 +201,14 @@ impl IntoConnectionInfo for ConnectionInfo {
     }
 }
 
+// Implementing IntoConnectionInfo for primitive string. Allowing a primitive string
```

Comment:
It's obvious where it's placed in the documentation that this is an impl of `IntoConnectionInfo`, so I would just start with the examples, i.e., 

```suggestion
/// Examples of valid strings:
```

By the way, if you want to preview it locally, you can just do this to view it in a browser:
```
cargo doc --all-features --open
```